### PR TITLE
Feat : userfocus completed

### PR DESCRIPTION
**Pull request by yogesh-**
Completed the userfocus feature which 
takes user input ->
displays as checkbox item -> 
if checked then strikethrough ->
a message

M
M1

### CODE REVIEW

**Review by syed-shaida-hussain — APPROVED** [review]
You can use variable names for colors in  CSS . Looks good to me for merging

**Comment by SRUJANA-PENUGONDA13** [comment]
Everything looks good. Below are few suggestions
 a. Remove console statements which you used for debugging
 b. In PR description you mentioned strikethrough the text once it got checked. But not able to find that code in this PR.

File: src/components/userfocus.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  const saveInput = (e) => {
+    if (e.key === "Enter") {
+      localStorage.setItem("user-focus", input);
+      window.location.reload();
+      console.log("saveInput event is getting fired");
+    }
+  };
+  const userfocus = localStorage.getItem("user-focus");
+  console.log("from userfocus", userfocus);
+
```
Comment (Sharmaryan) [inline src/components/userfocus.js:18]: remove console log

File: src/components/userfocus.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  const handleChecked = (e) => {
+    const randomNumber = Math.floor(Math.random() * 5);
+    const wishQuote = wishdb[randomNumber];
+    console.log(wishQuote);
+
```
Comment (Sharmaryan) [inline src/components/userfocus.js:23]: remove from here also

File: src/App.js [hunk 3]
```diff
@@ -82,13 +91,14 @@ function App() {
       ) : (
         <main>
           <p className="time">{time}</p>
-          <p>
+          <p className="font-color-default">
             {mycity}
             {"  "}
             {temp}
```
Comment (Sharmaryan) [inline src/App.js:97]: remove these {" "}

**Review by Sharmaryan — APPROVED** [review]
remove console logs, there are three or four, remove these brackets {" "}  and your UI is looking nice

**Comment by yogesh-** [comment]
> Everything looks good. Below are few suggestions a. Remove console statements which you used for debugging b. In PR description you mentioned strikethrough the text once it got checked. But not able to find that code in this PR.

yes made the commit shortly after raising the PR.

File: src/App.js (hunk 3 above)
Comment (yogesh-) [reply src/App.js:97]: ok

**Pull request merged**